client/src/views/About.vue

```diff
@@ -128,12 +151,46 @@ export default {
     drawCard() {
       unoService.drawCard(this.game_id, this.username)
         .then(this.updateData());
-    }
+    },
+    orgOff() {
+      if (this.sortByColor == true || this.sortByNum == true) {
+        this.loadingHand = true;
+      }
+      this.sortByNum = false;
+      this.sortByColor = false;
+    },
+    orgByNum() {
+      this.sortByNum = true;
+      this.sortByColor = false;
+
+      var colors = { 'red': 0, 'blue': 1 , 'green': 2, 'yellow': 3 }
```

Comment:
colors can probably be const here

Reply:
I'm sure I could but I don't know if it matters. And this way I can decide the order I want.